cloudinit/config/cc_salt_minion.py

```diff
@@ -45,6 +45,11 @@
                 service_name: salt-minion
                 config_dir: /etc/salt
                 conf:
+                    # file_client: local
```

Comment:
Any reason these are commented? I think it'd be better for them to be uncommented.

Reply:
@TheRealFalcon, this is added only as an example of how salt-minion can work in masterless mode.